boards/FSM/Inc/Io/Io_WheelSpeedSensors.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <stm32f3xx_hal.h>
+
+/**
+ * @brief Initializes the left and right wheel speed sensors
```

Comment:
nit; no need for `@brief` for all javadoc

Reply:
Sounds good, I got rid of them